src/Groups/GroupDetails/GroupDetailsContainer.js

```diff
@@ -55,7 +76,7 @@ const GroupDetailsContainer = ({
 
   const removeGroup = useCallback(
     () => {
-      mutator.groupDetails.DELETE(group)
+      mutator.groupDetails.DELETE(groupData.groupDetails)
```

Comment:
isn't it better to specify explicitly `{id: groupData.groupDetails.id}` ? so you can depend clearly on that attribute

Reply:
any reason? stripes-connect expects record, why do I need to create "record" with one field? plus pk key can be different 
https://github.com/folio-org/stripes-connect/blob/master/RESTResource/RESTResource.js#L219